### ISSUE TITLE
chrome driver version

### ISSUE THREAD

**Issue by cheezypotatoes** [ultrafunkamsterdam/undetected-chromedriver#1593]
selenium.common.exceptions.WebDriverException: Message: unknown error: cannot connect to chrome at 127.0.0.1:64480
from session not created: This version of ChromeDriver only supports Chrome version 114
Current browser version is 117.0.5938.132

how to fix this?

**Comment by FlorianMgs** [comment]
Update your chromedriver: https://chromedriver.chromium.org/downloads

**Comment by 2-X** [comment]
yeah but the library downloads them normally for builds 115, so manually updating the chromedriver works but is a silly solution for the long term. we added code to the patcher to download the matching chromedriver version from https://googlechromelabs.github.io/chrome-for-testing/latest-versions-per-milestone-with-downloads.json. happy to make a PR if y'all would find this helpful

**Comment by bikeros44** [comment]
Locate the undetected-chromedriver file at C:\Users\USER\AppData\Roaming\undetected-chromedriver\ delete the file then 
run  : pip uninstall undetected_chromedriver
after run :  pip install undetected_chromedriver   it should fix the problem

**Comment by dlliang14** [comment]
> yeah but the library downloads them normally for builds under version 115, so it would be nice to not have to manually download the chromedrivers in the long term.
> 
> we added code to the patcher to download the matching chromedriver version from https://googlechromelabs.github.io/chrome-for-testing/latest-versions-per-milestone-with-downloads.json. happy to make a PR if y'all would find this helpful

excuse me. how to use this json? or how to manually set the path of manually downloaded chromedriver？

**Comment by dlliang14** [comment]
i delete the", version_main=117" which  will fail to download chromedriver.
then the problem solved.
